salt/docker/pm_container/init.sls

```diff
@@ -0,0 +1,36 @@
+{% set current_version = "0.0.6" %}
```

Comment:
deploying new version is simple as:
  1. merging changes to master
  2. bumping this number
  3. salt-call state.highstate on vis